tests/integration/test_script_runner.py

```diff
@@ -248,7 +248,17 @@ def delay_rerun(*args):
     IntegrationTestFixture(
         name="quickstart",
         user_flow_script="tests/integration/docusaurus/tutorials/quickstart/quickstart.py",
-        backend_dependencies=[],
+        backend_dependencies=[BackendDependencies.PANDAS],
```

Comment:
This was missing before - we use `read_csv` so I think we need the Pandas dep?

Reply:
Do we need to test the old quickstart in develop anymore?

Reply:
probably not but I don't think it hurts to keep around until we remove the old guide?